scripts/check-commit.js

```diff
@@ -8,7 +8,7 @@ const oldestAncestor = String(exec(`bash -c 'diff -u <(git rev-list --first-pare
 
 console.log(`Oldest ancestor SHA: ${oldestAncestor}`);
 
-const messages = oldestAncestor && String(exec(`git log --format=%s ${oldestAncestor}..HEAD | sed '/^[[:space:]]*\$/d'`)).trim().split('\n').reverse();
+const messages = oldestAncestor && String(exec(`git log --format=%s ${oldestAncestor}..HEAD | sed '/^[[:space:]]*$/d'`)).trim().split('\n').reverse();
```

Comment:
how is this not needed?

Reply:
Since the backslash is singular inside a template literal (well the same case also applies to ordinary strings), it results in the backslash being removed and so the `$` is not escaped (this is the same before and after my change). My change is to remove it as it'll be unescaped anyway.

If the `$` actually needs to be escaped then we'll have to change to double backslashes.

Reply:
ah, right - the intention was to escape it in the template literal. i wouldn’t want a { to show up in the future and have it be confused for an interpolation, which is why i left in the escape character - to document that intent. I think it’s better to leave it.

Reply:
I see, I'll leave it as-is then.